src/Console/Command/AddPrefixCommand.php

```diff
@@ -59,22 +61,29 @@ protected function configure()
             ->setName('add-prefix')
             ->setDescription('Goes through all the PHP files found in the given paths to apply the given prefix to namespaces & FQNs.')
             ->addArgument(
-                self::PREFIX_ARG,
-                InputArgument::REQUIRED,
-                'The namespace prefix to add'
-            )
-            ->addArgument(
                 self::PATH_ARG,
-                InputArgument::REQUIRED | InputArgument::IS_ARRAY,
+                InputArgument::IS_ARRAY,
                 'The path(s) to process.'
             )
             ->addOption(
-                self::OUTPUT_DIR,
+                self::PREFIX_OPT,
+                'p',
+                InputOption::VALUE_REQUIRED,
+                'The namespace prefix to add'
+            )
+            ->addOption(
+                self::OUTPUT_DIR_OPT,
                 'o',
                 InputOption::VALUE_REQUIRED,
                 'The output directory in which the prefixed code will be dumped.',
                 'build'
             )
+            ->addOption(
+                self::FORCE_OPT,
+                'f',
+                InputOption::VALUE_NONE,
+                'Deletes any existing content in the output directory without any warning'
```

Comment:
Insert `rm -rf /` joke ;).

Do we test that the output directory is removed somewhere (esp. for Windows where Symfony Filesystem appears to have issues with this)?